build_tests/app.py

```diff
@@ -0,0 +1,66 @@
+import uuid
+
+from beta9 import Image, function
+
+numpy_311_img = (
+    Image(python_version="python3.11")
+    .add_python_packages(["numpy"])
+    .add_commands([f"echo {uuid.uuid4()}"])
+)
+ffmpeg_img = Image().add_commands(
+    ["apt-get update && apt-get install -y ffmpeg", f"echo {uuid.uuid4()}"]
+)
+nv_cuda_1222_img = Image(
+    python_version="python3.10",
+    base_image="nvidia/cuda:12.2.2-cudnn8-runtime-ubuntu22.04",
+).add_commands([f"echo {uuid.uuid4()}"])
+nv_cuda_1222_torch_img = (
+    Image(
+        python_version="python3.10",
+        base_image="nvidia/cuda:12.2.2-cudnn8-runtime-ubuntu22.04",
+    )
+    .add_python_packages(["torch"])
+    .add_commands([f"echo 1-{uuid.uuid4()}"])
+)
+rapids_img = Image(base_image="rapidsai/rapidsai").add_commands(
+    [f"echo {uuid.uuid4()}"]
+)
```

Comment:
Would be nice to do this on a matrix of python versions too, but I can see that not being desirable for cost of storage reasons.

Reply:
Yeah it would be nice to cover a wider variety of bases. I think I'll add those and then flag them as local. That way you could do `make build-test local` and that will run extra tests and exclude ones that will break on dev machines due to architecture.